src/Behat/Testwork/Exception/FatalThrowableError.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Behat Testwork.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Testwork\Exception;
```

Comment:
This exception should be in ``Behat\Testwork\Call\Exception``, as it is only used by the Call subsystem